cli/internal/kubernetes/upgrade.go

```diff
@@ -347,6 +348,30 @@ func (u *Upgrader) BackupConfigMap(ctx context.Context, name string) error {
 	return nil
 }
 
+// UpdateAttestationConfig fetches the cluster's attestation config, compares them to a new config,
+// and updates the cluster's config if it is different from the new one.
+func (u *Upgrader) UpdateAttestationConfig(ctx context.Context, newAttestConfig config.AttestationCfg) error {
+	// backup of previous measurements
+	joinConfig, err := u.stableInterface.GetConfigMap(ctx, constants.JoinConfigMap)
+	if err != nil {
+		return fmt.Errorf("getting join-config configmap: %w", err)
+	}
+	joinConfig.Data[constants.AttestationConfigFilename+"_backup"] = joinConfig.Data[constants.AttestationConfigFilename]
```

Comment:
```suggestion
```
We already created a backup of the config in a new ConfigMap in a separate call previously